src/test/scala/org/broadinstitute/dsde/agora/server/AgoraTestData.scala

```diff
@@ -97,7 +98,93 @@ object AgoraTestData {
+                               |          "typeName": "String"
+                               |        }
+                               |      },
+                               |      "typeDisplayName": "String?",
+                               |      "optional": true,
+                               |      "default": null
+                               |    },
+                               |    {
+                               |      "name": "grep.pattern",
+                               |      "valueType": {
+                               |        "typeName": "String"
+                               |      },
+                               |      "typeDisplayName": "String",
+                               |      "optional": false,
+                               |      "default": null
+                               |    },
+                               |    {
+                               |      "name": "input_files",
+                               |      "valueType": {
+                               |        "typeName": "Array",
+                               |        "arrayType": {
+                               |          "typeName": "File"
+                               |        }
+                               |      },
+                               |      "typeDisplayName": "Array[File]",
+                               |      "optional": false,
+                               |      "default": null
+                               |    }
+                               |  ],
+                               |  "outputs": [
+                               |    {
+                               |      "name": "grep.out",
+                               |      "valueType": {
+                               |        "typeName": "Array",
+                               |        "arrayType": {
+                               |          "typeName": "File"
+                               |        }
+                               |      },
+                               |      "typeDisplayName": "Array[File]"
+                               |    },
+                               |    {
+                               |      "name": "wc.count",
+                               |      "valueType": {
+                               |        "typeName": "Int"
+                               |      },
+                               |      "typeDisplayName": "Int"
+                               |    }
+                               |  ],
+                               |  "images": [],
+                               |  "submittedDescriptorType": {
+                               |    "descriptorType": "WDL",
+                               |    "descriptorTypeVersion": "draft-2"
+                               |  },
+                               |  "importedDescriptorTypes": [],
+                               |  "meta": {}
+                               |}""".stripMargin
+
```

Comment:
would it make more sense to save these payloads as json files in src/test/resources somewhere, and read them into lazy vals? Saves a bunch of annoying Scala code here.

Reply:
Good idea -- I didn't do this for everything in that file b/c I think many of those existing tests should be re-examined, but I did it for everything used in this PR and set up the approach that should be easy to extend